frontend/components/Layouts/BlogPageLayout.tsx

```diff
@@ -0,0 +1,88 @@
+import { format } from 'date-fns'
+import { getUTCDate } from '../../utils'
+
+type Props = {
+  date: string
+  title: string
+  image: string
+  children: React.ReactNode
+}
+
```

Comment:
I'd like to use interfaces for defining simple objects and in particular props 😊 This is also in the style guide but I'd like to write more there shortly 😄

Reply:
Oh I think I may have done that before seeing the style guide. I read somewhere that types are better to use as a default than interfaces, but it seems like interfaces are the most popular way with React Props, so let's go with interfaces! 👍

Reply:
Sweet! 😄